src/sagemaker/serializers.py

```diff
@@ -241,3 +241,27 @@ def serialize(self, data):
         buffer = io.BytesIO()
         scipy.sparse.save_npz(buffer, data)
         return buffer.getvalue()
+
+
+class LibSVMSerializer(BaseSerializer):
+    """Searilize data of various formats to a LibSVM-formatted string."""
+
+    CONTENT_TYPE = "text/libsvm"
+
+    def serialize(self, data):
```

Comment:
Do users frequently use CSV files, NumPy arrays, or pandas data frames with XGBoost? If so, it may be worthwhile serializing CSV files, NumPy arrays, or pandas data frames to LibSVM, especially if `LibSVMSerializer` is the default serializer for XGBoost.

Reply:
Users often use CSV files, but everything including CSV and libsvm gets converted to XGBoost's DMatrix. This happens on the container side. I added a note in the docstring that the data must already be in libsvm format.